modules/delegates/components/DelegateParticipationMetrics.tsx

```diff
@@ -0,0 +1,54 @@
+/** @jsx jsx */
+import BigNumber from 'bignumber.js';
+import { Box, Text, jsx } from 'theme-ui';
+import { Delegate } from '../types';
+
+export function DelegateParticipationMetrics({ delegate }: { delegate: Delegate }): React.ReactElement {
+  const styles = {
+    row: {
+      display: 'flex',
+      justifyContent: 'space-between',
+      marginTop: 1,
+      marginBottom: 1
+    },
+    text: {
+      color: 'secondaryAlt',
+      fontWeight: 'semiBold',
+      fontSize: 3
+    }
+  };
+  return (
+    <Box p={[3, 4]}>
+      <Text as="p" sx={{
+        fontSize: '18px',
+        fontWeight: 500,
+        mb: 3
+      }}>Participation breakdown</Text>
+
+      <Box sx={styles.row}>
+        <Text as="p" sx={styles.text}>
+          Participation
+        </Text>
+        <Text as="p" sx={styles.text} ml={2}>
+          {delegate.combinedParticipation ?? 'Untracked'}
+        </Text>
+      </Box>
+      <Box sx={styles.row}>
+        <Text as="p" sx={styles.text}>
+          Communication
+        </Text>
+        <Text as="p" sx={styles.text} ml={2}>
+          {delegate.communication ?? 'Untracked'}
+        </Text>
+      </Box>
+      <Box sx={styles.row}>
+        <Text as="p" sx={styles.text}>
+          MKR Delegated
```

Comment:
`MKR Delegated` isn't one of the participation metrics. I do think we should show this on the credentials page.

Design:
<img width="776" alt="Screen Shot 2021-08-30 at 11 54 26 AM" src="https://user-images.githubusercontent.com/5225766/131321950-1e50b63d-a8d7-433b-b070-b310a6fae999.png">